examples/terraform/Earthfile

```diff
@@ -0,0 +1,19 @@
+
+build:
+    FROM hashicorp/terraform:light
+
+    ARG AWS_ACCESS_KEY_ID
+    ARG AWS_SECRET_ACCESS_KEY
```

Comment:
We should probably add a comment here saying that in prod environments we should use a secret. However in this case this is not really a secret so it's ok.

Reply:
Yeah, I was also looking at some of the terraform and some comments are probably in order.